scripts/launch/mainnet/LaunchOmnibusMainnet.sol

```diff
@@ -0,0 +1,466 @@
+            });
+        }
+
+        // AllowedTokensRegistry Roles Transition
+        {
+            bytes32 DEFAULT_ADMIN_ROLE = bytes32(0);
+            bytes32 ADD_TOKEN_TO_ALLOWED_LIST_ROLE = keccak256("ADD_TOKEN_TO_ALLOWED_LIST_ROLE");
+            bytes32 REMOVE_TOKEN_FROM_ALLOWED_LIST_ROLE = keccak256("REMOVE_TOKEN_FROM_ALLOWED_LIST_ROLE");
+
+            voteItems[index++] = VoteItem({
+                description: "39. Grant DEFAULT_ADMIN_ROLE to Voting on AllowedTokensRegistry",
+                call: _forwardCall(
+                    AGENT, ALLOWED_TOKENS_REGISTRY, abi.encodeCall(IOZ.grantRole, (DEFAULT_ADMIN_ROLE, VOTING))
+                )
+            });
+
+            voteItems[index++] = VoteItem({
+                description: "40. Revoke DEFAULT_ADMIN_ROLE from Agent on AllowedTokensRegistry",
+                call: _votingCall(ALLOWED_TOKENS_REGISTRY, abi.encodeCall(IOZ.revokeRole, (DEFAULT_ADMIN_ROLE, AGENT)))
+            });
+
+            voteItems[index++] = VoteItem({
+                description: "41. Revoke ADD_TOKEN_TO_ALLOWED_LIST_ROLE from Agent on AllowedTokensRegistry",
+                call: _votingCall(
+                    ALLOWED_TOKENS_REGISTRY, abi.encodeCall(IOZ.revokeRole, (ADD_TOKEN_TO_ALLOWED_LIST_ROLE, AGENT))
+                )
+            });
+
+            voteItems[index++] = VoteItem({
+                description: "42. Revoke REMOVE_TOKEN_FROM_ALLOWED_LIST_ROLE from Agent on AllowedTokensRegistry",
+                call: _votingCall(
+                    ALLOWED_TOKENS_REGISTRY, abi.encodeCall(IOZ.revokeRole, (REMOVE_TOKEN_FROM_ALLOWED_LIST_ROLE, AGENT))
+                )
+            });
+        }
+
+        // WithdrawalVault admin transition
+        {
+            voteItems[index++] = VoteItem({
+                description: "43. Set admin to Agent on WithdrawalVault",
+                call: _votingCall(WITHDRAWAL_VAULT, abi.encodeCall(IWithdrawalVaultProxy.proxy_changeAdmin, (AGENT)))
+            });
+        }
+
+        // InsuranceFund owner transition
+        {
+            voteItems[index++] = VoteItem({
+                description: "44. Set owner to Voting on InsuranceFund",
+                call: _forwardCall(AGENT, INSURANCE_FUND, abi.encodeCall(IInsuranceFund.transferOwnership, (VOTING)))
+            });
+        }
+
+        // Validate transferred roles
+        {
+            voteItems[index++] = VoteItem({
+                description: "45. Validate transferred roles",
+                call: _votingCall(ROLES_VALIDATOR, abi.encodeCall(IRolesValidator.validateVotingLaunchPhase, ()))
+            });
+        }
+
+        // Submit first dual governance proposal
+        {
+            ExternalCallsBuilder.Context memory dgProposalCallsBuilder =
+                ExternalCallsBuilder.create({callsCount: DG_PROPOSAL_CALLS_COUNT});
+
+            // 1. Execution is allowed before Tuesday, 1 July 2025 00:00:00
```

Comment:
Jun 25 + 5 Aragon voting days + 3 after submission delay + 1 after schedule delay is 4 July. I suggest to set deadline to 11th July to be sure